cli/azd/extensions/azure.ai.agents/internal/cmd/init.go

```diff
@@ -258,17 +266,22 @@ func ensureProject(ctx context.Context, azdClient *azdext.AzdClient) (*azdext.Pr
 	return projectResponse.Project, nil
 }
 
-func ensureEnvironment(ctx context.Context, azdClient *azdext.AzdClient) (*azdext.Environment, error) {
+func ensureEnvironment(ctx context.Context, flags *initFlags, azdClient *azdext.AzdClient) (*azdext.Environment, error) {
 	envResponse, err := azdClient.Environment().GetCurrent(ctx, &azdext.EmptyRequest{})
 	if err != nil {
 		fmt.Println("Lets create a new default environment for your project.")
 
+		envArgs := []string{"env", "new"}
+		if flags.env != "" {
+			envArgs = append(envArgs, flags.env)
+		}
+
```

Comment:
Is the intended behaviour that if you already have an environment, the `flags.env` value is ignored? I was expecting `init` to create `flags.env` regardless